### PR TITLE
Support Symfony 4.4

### PR DESCRIPTION
**Pull request by gekomod**
Remove and update code

### CODE REVIEW

File: src/Mert/TicketBundle/Form/TicketType.php [hunk 1]
```diff
@@ -5,21 +5,26 @@
...
 class TicketType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
 
-        $builder->add('category', 'entity', array('class' => 'Mert\TicketBundle\Entity\Category'));
-        $builder->add('title', 'text', array('label' => false));
-        $builder->add('content', 'textarea', array('label' => false));
-        $builder->add('priority', 'choice',
+        $builder->add('category', EntityType::class, array('class' => 'Mert\TicketBundle\Entity\Category'));
```
Comment (mertoksuz) [inline src/Mert/TicketBundle/Form/TicketType.php:19]: Thanks for the support, only i see, you are removing this entity but still using in here.

**Review by mertoksuz — CHANGES_REQUESTED** [review]

**Comment by mertoksuz** [comment]
Why we are removing the User entity ?

**Comment by gekomod** [comment]
User Entity is deleted a double file which has the extension "php ~" the original file

File: src/Mert/TicketBundle/Form/TicketType.php (hunk 1 above)
Comment (gekomod) [reply src/Mert/TicketBundle/Form/TicketType.php:19]: User Entity is deleted a double file which has the extension "php ~" the original file


```
$builder->add('category', EntityType::class, array('class' => 'Mert\TicketBundle\Entity\Category'));
```
This is read Entity Category

Comment (mertoksuz) [reply src/Mert/TicketBundle/Form/TicketType.php:19]: Ah, i see now, thank you

**Review by mertoksuz — APPROVED** [review]
Thanks for support

**Pull request merged**